portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,176 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>My Portfolio 1</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital@1&display=swap" rel="stylesheet">
+    
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <nav> 
+          <span>Navbar</span>
+            <a href="#box">Color Picker</a>
+            <a href="#gallery">Gallery</a>
+            <a href="#aboutContent">About</a>
+      </nav>
+
+    <div class="hero"> 
+        <div class="hero-text"> 
+            <h1>Sofia Montana</h1>
+            
+        </div>
     </div>
+
+    <div id="aboutContent"> 
+        <h2 style="font-size: 36px;">About Me... </h2>
+        <p style="font-size: 20px;"> 
+            Hello, I am Sofia Montana De Bedout, I am twenty year old 
+            university student from Miami, Florida. I really like Computer Science and 
+            am really excited to learn more about this field. My family and I are orginally 
+            from Bogota, Colombia and I am fully bilingual in Spanish and English. I love 
+            playing video games, watching Netflix and playing sports. Also, I am quite the Math 
+            nerd I love Math and have a job tutoring students in Math at my university.
+        </p>
+    </div>
+
+    <!-- place the cool image hover effect of each (for each) -->
+    <div id="gallery">
```

Comment:
Use a consistent style for choosing between whether to use an "id" or a "class".

Typically, id is used when the element is unique, and class is used when there will be several elements reusing that class.

In practice at google, we virtually never use "id" and only use "class".

Reply:
ok noted I just thought that using class instead of id in this case would be weird because I am only using  in this one instance.